src/main/java/org/kuali/kfs/module/purap/document/web/struts/PurchasingActionBase.java

```diff
@@ -1659,11 +1661,13 @@ public ActionForward insertBONote(ActionMapping mapping, ActionForm form, HttpSe
 						KRADConstants.NOTE_TOPIC_TEXT_PROPERTY_NAME), CUPurapKeyConstants.ERROR_ADD_NEW_NOTE_SEND_TO_VENDOR_NO_ATT);
 				return mapping.findForward(KFSConstants.MAPPING_BASIC);
 			} else {
+                String fileSizeLimitParameterValue = getMaxPoSendToVendorFileSizeParameterValue();
 				if (isAttachmentSizeExceedSqLimit(form, "add")) {
 					return mapping.findForward(KFSConstants.MAPPING_BASIC);
-				} else if (attachmentFile.getFileSize() > SIZE_5MB) {
-					GlobalVariables.getMessageMap().putError(String.format("%s.%s",KRADConstants.NEW_DOCUMENT_NOTE_PROPERTY_NAME,
-							KRADConstants.NOTE_TOPIC_TEXT_PROPERTY_NAME), CUPurapKeyConstants.ERROR_ATT_FILE_SIZE_OVER_LIMIT, attachmentFile.getFileName(), "5");
+				} else if (attachmentFile.getFileSize() > FileUtil.getBytes(fileSizeLimitParameterValue)) {
+                    GlobalVariables.getMessageMap().putError(String.format("%s.%s",KRADConstants.NEW_DOCUMENT_NOTE_PROPERTY_NAME,

```

Comment:
Some of the edited lines' whitespace formatting seems a bit off.

Reply:
After some recent commits, looking at the file on the link below the commits made seem to have improved the formatting related . When viewing the method in the IDEs, the formatting looks correct (does it on your end?). However in the link below (viewing in GitHub) the code within the method (and elsewhere in the file) seems to have undesirable formatting. 

- Should we try to fix the formatting in the rest of the method or create another user story for the whole method/file?

https://github.com/CU-CommunityApps/cu-kfs/blob/027b8c49dc72670f1ae8fbd098be6c71668c0753/src/main/java/org/kuali/kfs/module/purap/document/web/struts/PurchasingActionBase.java

Reply:
The formatting differences seem to be caused by the file using tabs in most places instead of spaces, where the file was likely created before our team standardized on the use of spaces. (Our IDEs might only be indenting tabs by 4 units even though the GitHub UI indents them by more than that, which is why the formatting may look correct in the IDE but not in GitHub.)

If it ends up being too complex to clean up this section of whitespace at this time, then I'm fine with deferring the whitespace cleanup to a separate user story.

Reply:
I think that it would be better to do it in a separate user story so it is slightly easier to discern what actually changed. I tried to fix the method but GitHub didn't show that was only indentation.